src/main/java/Nanami.java

```diff
@@ -0,0 +1,289 @@
+package src.main.java;
+
+import src.main.java.task.Deadline;
+import src.main.java.task.Event;
+import src.main.java.task.Task;
+import src.main.java.task.ToDo;
+import src.main.java.taskExceptions.DeadlineMismatchedParameterException;
+import src.main.java.taskExceptions.EmptyTaskException;
+import src.main.java.taskExceptions.EventMismatchedParameterException;
+import src.main.java.taskExceptions.ToDoMismatchedParameterException;
+
+import java.io.*;
+import java.util.Arrays;
+
+public class Nanami {
+    private static Task[] tasklist = new Task[100];
+    private static int taskcount;
+    private int taskCount = 0;
+    private static boolean applicationOpen = true;
+    private static String filepath = "taskDrive.txt";
+    private static File outfile = new File(filepath);
+    private static FileWriter overwriter, appender;
+
+    public static void main(String args[]) {
+        // driver code to begin the application
+        loadData();
+        // clear the screen of loading data
+        System.out.print("\033[H\033[2J");
+        System.out.flush();
+
+        giveIntroduction();
+
+        while (applicationOpen) {
+            String input = readUserInput().trim();
+            String [] commandWords = input.split("\\s+"); // cuts phrases at spaces into an array
+
+            if (input.equals("")) { // check for empty string
+                System.out.println("You gotta give me something to work with here.");
+            }
+
+            // one-word commands
+            if (input.equals("bye") || input.equals("Bye")) {
```

Comment:
not a coding standard/quality issue here (this is alright) but a suggested library you can use is input.toLowerCase().equals("bye") :] - so you can just convert everything to lowercase and check just one case (e.g. "bye")!